.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -19,7 +19,13 @@ assignees: ''
 
 #### Screenshot
 
-#### LMMS version used
+#### Affected LMMS versions
+
+<!--
+LMMS's version is visible on the splash screen and under Help > About.
```

Comment:
I'm not a native English, but I'd have avoided the double `s`. Feel free to ignore.
```suggestion
LMMS' version is visible on the splash screen and under Help > About.
```

Reply:
What I learned is that you only drop the 's' in a possessive for plurals. E.g. "Texas's taxes' effects", but I could change it if you'd like as I think this varies by guide/school/whatever.

Reply:
Actually, since you said "feel free to ignore" I'll just roll with it as is haha

Reply:
Sorry, I really had to ask in discord. "LMMS's" is correct. "LMMS'" can only be used in "slang", but it's not really good.

Resolving the comment.